main.py

```diff
@@ -378,6 +380,27 @@ def get_metadata(self, circuit_id):
         except KeyError as error:
             raise NotFound(f"circuit_id {circuit_id} not found.") from error
 
+    @rest("v2/evc/metadata", methods=["POST"])
+    @validate(spec)
+    def bulk_add_metadata(self, data):
+        """Add metadata to a bulk of EVCs."""
+        circuit_ids = data.pop("circuit_ids", None)
+        if circuit_ids is None:
```

Comment:
Really nice how you deferred most of the validation to the spec. There's room to also get ride of this `BadRequest` here you can specify a `requestBody`, defining a schema component with `circuit_ids`, can you also augment this part?